packages/collector/Collector_Db.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+function collector_esc_sql_identifier($tableName) {
+	return str_replace('`', '``', esc_sql($tableName));
+}
+
+function collector_dump_db($zip)
+{
+	$tables   = collector_get_db_tables();
+	$sql_dump = '';
+
+	$sql_dump .= sprintf("-- %s\n", json_encode(['SECTION START' => 'SCHEMA']));
+
+	foreach($tables as $table)
+	{
+		$sql_dump .= sprintf("-- %s\n", json_encode(['ACTION' => 'DROP', 'TABLE' => $table]));
+		$sql_dump .= sprintf("DROP TABLE IF EXISTS `%s`;\n", collector_esc_sql_identifier($table));
+		$sql_dump .= sprintf("-- %s\n", json_encode(['ACTION' => 'CREATE', 'TABLE' => $table]));
+		$sql_dump .= preg_replace("/\s+/", " ", collector_dump_db_schema($table)) . "\n";
+	}
+
+	$sql_dump .= sprintf("-- %s\n", json_encode(['SECTION END' => 'SCHEMA']));
+	$sql_dump .= sprintf("-- %s\n", json_encode(['SECTION START' => 'RECORDS']));
+
+	global $wpdb;
+
+	// Process in reverse order so wp_users comes before wp_options
+	// meaning the fakepass will be cleared before transients are
```

Comment:
If the ordering is important here, let's make sure `$tables` are sorted. At the moment the seem to be no guarantees about what comes first. However, I'm not sure what clears the fakepass as the comment mentions, why is that important, and what is the relation to transients.

Reply:
I didn't find a reason why this would be needed.